pkg/cmd/cli/secrets/add_secret_to_obj.go

```diff
@@ -32,6 +32,11 @@ To use your secret inside of a pod or as a push, pull, or source secret for a bu
 To use your secret as an image pull secret, you must add a 'pull' secret to your service account like this:
 
   $ %s serviceaccount/sa-name secrets/secret-name --for=pull
```

Comment:
use `%[1]s` for the command name, then you only need to pass the name to sprintf once